doc/application/index.rst

```diff
@@ -645,6 +645,29 @@ Zephyr binary into your application directory.
 You can also define the ``SOC_ROOT`` variable in the application
 :file:`CMakeLists.txt` file.
 
+DeviceTree Definitions
+======================
+
+Additional DeviceTree directory tree's, or DTS_ROOT's, can be added by
```

Comment:
When the word is plural you don't use an apostrophe:
```suggestion
Additional DeviceTree directory trees, or DTS_ROOTs, can be added by
```

Reply:
Yeah, this I should have known ...